docs/user-guide/dsl.md

```diff
@@ -80,6 +103,8 @@ Like with all domain-specific languages, __the number of things you can do insid
 - Iterate over the results of an invocation to parallelize the execution of a node.
 - Add all the results of a partitioned node in a list and pass that list to a fan-in node.
 - Return multiple outputs from the DAG by returning a dictionary of `#!python str -> node_output`.
+- Parameter defaults for tasks, e.g. `#!python def my_task(a, b=2)` will use the default value `2` if `b` is not passed. The same is true for DAG parameters.
+- Pass hardcoded values to a DAG's task or another DAG, e.g. when a nested DAG has two parameters and one is hardcoded in the main DAG.
```

Comment:
This line is not necessary. This functionality was already there in the second bullet point :)

Reply:
done, resolving